### DISCUSSION TITLE
Manual Flow Trigger From Insights Dashboard

### DISCUSSION THREAD

**Discussion by phazonoverload** [directus/directus#18864] [Feature Requests]
### Summary

Flows with a manual trigger are currently shown on collection and item pages. When on a collection page, zero or more items are passed into the trigger. When on an item page, the item is passed into the trigger. 

This feature request would allow for Flows with manual triggers to be executed from an insights dashboard - either from the right-hand pane or with a new panel type that contains 1+ buttons (similar to Apple Shortcut Widgets). 

### Motivation

All insights panels provided with Directus are read-only representations or summaries of data. Flows allow for data creation/updates/deletions. This feature request brings these two parts of Directus closer together so users can act while seeing data in a dashboard. 

### Detailed Design

Option 1: like the current implementation, allow flows with manual triggers to be shown in specific insights dashboards.

Option 2: a new panel type allows for 1+ flows to be triggered when their button is clicked. 

### Must Have:
- A button that allows a flow to be triggered

### Could Have:
- A way to insert data shown in the dashboard into the flow trigger.

### Drawbacks

Currently, it is pretty clear which context the manual flow triggers are running with - either because it is a single item page, or because of items explicitly checked from a collection layout.

### Alternatives

None at this time.

### Adoption Strategy

This is a progressive enhancement on the existing application.

**Comment by LevelUpGit** [comment]
Also it would be good that flows can triggering by custom interface button.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
